test/unit/plugins/ucx/ucx_worker_test.cpp

```
@@ -42,6 +42,72 @@ checkCudaError(cudaError_t result, const char *message) {
+#endif
+}
+
+static inline void gpuMemcpyD2H(void *dst, const void *src, size_t size, const char *message) {
+#ifdef HAVE_CUDA
+    checkCudaError(cudaMemcpy(dst, src, size, cudaMemcpyDeviceToHost), message);
+#elif defined(HAVE_ROCM)
+    checkHipError(hipMemcpy(dst, src, size, hipMemcpyDeviceToHost), message);
+#endif
+}
+
+static inline void gpuMemcpyH2D(void *dst, const void *src, size_t size, const char *message) {
```

Cppcheck (2.21.0)

[style] The function 'gpuMemcpyH2D' is never used.

(unusedFunction)